charts/nextcloud/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: nextcloud
-version: 6.6.10
+version: 6.7.0
```

Comment:
After the release of 7.0+ can you kindly update the version and check against the new release?